### PR TITLE
Build TextAnalytics REST Binding

### PR DESCRIPTION
**Pull request by jianghaolu**

### CODE REVIEW

File: textanalytics/low-level-dynamic/azure-ai-textanalytics-models/src/samples/java/com/azure/ai/textanalytics/RecognizeLinkedEntitiesWithModels.java [hunk 1]
```diff
@@ -33,9 +33,9 @@ public static void main(String[] args) {
                 .setId("0").setText("Old Faithful is a geyser at Yellowstone Park.");
         MultiLanguageBatchInput batchInput = new MultiLanguageBatchInput().setDocuments(Collections.singletonList(input));
 
-        EntityLinkingResult result = client.entitiesLinking() // DynamicRequest
+        EntityLinkingResult result = client.getLinkedEntities() // DynamicRequest
                 .setBody(batchInput) // DynamicRequest
-                .context(Context.NONE) // DynamicRequest
+                .setContext(Context.NONE) // DynamicRequest
```
Comment (JonathanGiles) [inline textanalytics/low-level-dynamic/azure-ai-textanalytics-models/src/samples/java/com/azure/ai/textanalytics/RecognizeLinkedEntitiesWithModels.java]: Don't we plan to remove the setContext method?

Comment (srnagar) [reply textanalytics/low-level-dynamic/azure-ai-textanalytics-models/src/samples/java/com/azure/ai/textanalytics/RecognizeLinkedEntitiesWithModels.java]: Yeah, based on offline discussion, we decided to have `send()`, `send(Context context)` and `sendAsyc()` (async uses Reactor context).

File: textanalytics/low-level-dynamic/azure-ai-textanalytics/README.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+### Prerequisites
+
+### Authenticate the client
+
+To create the client, you can use an `AzureKeyCredential`, with your API Key:
+
+```java
+TextAnalyticsClient client = new TextAnalyticsClientBuilder()
+        .credential(new AzureKeyCredential("{api key from portal}"))
```
Comment (srnagar) [inline textanalytics/low-level-dynamic/azure-ai-textanalytics/README.md:19]: For the purpose of this user study, I think we should short-circuit this and store the API key in env var and read from there. Users spend quite a bit of time trying to locate the api key on the portal.

Comment (jianghaolu) [reply textanalytics/low-level-dynamic/azure-ai-textanalytics/README.md:19]: Yes that's what the user study is doing: https://github.com/jianghaolu/low-level-java-proto/pull/2/files#diff-a1d4bdc1088c7eb613ef35ed82667215e643dffdd3054c7693ec54da0ab5523fR9

**Pull request merged**